app/templates/signatures/ongoing.hbs

```diff
@@ -1,104 +1,120 @@
+              @selectedItems={{this.selectedStatuses}}
+              @didUpdate={{this.onChangeStatus}}
               />
-            {{/if}}
+          </div>
+          {{#if this.currentSession.isAdmin}}
+          <div class="au-c-card au-o-box au-o-box--small">
+              <Search::MinisterFilter
+                @selected={{this.mandatees}}
+                @onChange={{this.setMandatees}}
+                @hidePastMinistersOption={{true}}
+                @label={{"submitter"}}
+              />
+          </div>
           {{/if}}
-        </td>
-        <td>
-          <AuPill @size="small">
-            {{#let piece.documentContainer.type as |type|}}
-              {{#if type}}
-                {{type.label}}
-              {{else}}
-                {{t "no-document-type"}}
-              {{/if}}
+        </div>
+      </div>
+    </div>
+  </m.sidebar>
+  <m.content @scroll={{true}}>
+    {{#if this.model.length}}
+      <KDataTable
+        data-test-route-search-publication-data-table
+        @hoverable={{true}}
+        @striped={{true}}
+        @loading={{this.isLoadingModel}}
+        @content={{@model}}
+        @page={{this.page}}
+        @size={{this.size}}
+        @numberOfItems={{@model.length}}
+        @totalNumberOfItems={{@model.meta.count}}
+        @onChangeSize={{fn (mut this.size)}}
+        @onChangePage={{fn (mut this.page)}}
+        @onClickRow={{this.navigateToPublicationFlow}}
+        >
+        <:header>
+          <th class="auk-table__col--6">{{t "document-name"}}</th>
+          <th class="auk-table__col--1">{{t "document-type"}}</th>
+          <th class="auk-table__col--1">{{t "ministers"}}</th>
+          <th class="auk-table__col--2">{{t "date-approved"}}</th>
+          <th class="auk-table__col--2">{{t "status"}}</th>
+          <th class="auk-table__col--1"></th>
+          </:header>
+          <:body as |signFlow|>
+            {{#let signFlow.signSubcase.signMarkingActivity as |signMarkingActivity|}}
+              {{#let signMarkingActivity.piece as |piece|}}
+                <td>
+                  {{#if piece}}
+                    {{piece.name}}
+                    {{#if (this.isConfidential piece.accessLevel)}}
+                      <AccessLevelPill
+                        @accessLevel={{piece.accessLevel}}
+                        @isEditable={{false}}
+                      />
+                    {{/if}}
+                  {{/if}}
+                </td>
+                <td>
+                  <AuPill @size="small">
+                    {{#let piece.documentContainer.type as |type|}}
+                      {{#if type}}
+                        {{type.label}}
+                      {{else}}
+                        {{t "no-document-type"}}
+                      {{/if}}
+                    {{/let}}
+                  </AuPill>
+                </td>
+              {{/let}}
+              <td>
+                {{#let (await (this.getMandateeNames signFlow)) as |names|}}
```

Comment:
Not that important, but there is no way for a user to differentiate between "loading" and "no minister" here. Might be useful to display a loader while the mandatees are loading, like we do for the status pill